pam_authz/pam/README.md

```diff
@@ -0,0 +1,64 @@
+# Styra's PAM authorization module
+
+This implements a PAM module that integrates with Open Policy Agent (OPA)
+to make authorization decisions. When invoked, the module queries OPA
+and if no errors are returned, then the action is allowed. This allows
+administrators to implement fine-grained access control. For
+example, one can use this module to limit usage of `sudo` to particular
+times of day.
+
+This module is not intended to perform authentication. One can use standard
+linux accounts for that or a user database such as LDAP.
+
+
+## Usage:
+
+This repo includes a detailed [example usage of the pam_authz module](../docker/README.md),
```

Comment:
I'm referring to the docs. I'd imagine there'll be some overlap/duplication required as the docs/example/tutorial should probably have the docker-compose file inlined for ease of use.